spec/policies/idv/flow_policy_spec.rb

```diff
@@ -68,7 +67,6 @@
         expect(idv_session.had_barcode_attention_error).to be_nil
 
         expect(idv_session.ssn).to be_nil
-        expect(idv_session.threatmetrix_session_id).to be_nil
```

Comment:
should we assert that it is not nil?

Reply:
We can, though with this change the `threatmetrix_sesison_id` is no longer something the `FlowPolicy` cares about. Setting it and adding an assertion that it is not nil may be confusing with the context of this change.

Reply:
ok would it make sense in the context of preserving the value to add a spec elsewhere in the PR?

Reply:
I added a test in aad6079c3e58b64e14c68b7fb2a7c220de62c9c2